ext/winevt/winevt_utils.cpp

```diff
@@ -267,7 +267,12 @@ extract_user_evt_variants(PEVT_VARIANT pRenderedValues, DWORD propCount)
         rb_ary_push(userValues, rbObj);
         break;
       case EvtVarTypeHexInt64:
-        rbObj = rb_sprintf("%#I64x", pRenderedValues[i].UInt64Val);
```

Comment:
Ruby's `rb_sprintf` does not support `%#I64x` format string.
We should use hand written format code here: https://github.com/fluent-plugins-nursery/winevt_c/pull/35/files#diff-7d3a7f81db65ff315b6f1e9e118b13ac952f66cc81fae2f05b10c8c6a4c69ac8R270-R275